### PR TITLE
Added clarification to the AC governance

### PR DESCRIPTION
**Pull request by rkoshak**
I added clarification that the clock doesn't start to tick on a vote until there is actually a call for a vote. 

Added a section to describe how the AC can generate an idea and elicit feedback and discussion from the community at large. This section also discusses how a member of the community can bring the attention of the AC to a topic that may need AC input.

Solves #895

Signed off by Richard Koshak <rlkoshak@gmail.com>

### CODE REVIEW

File: developers/contributing/governance.md [hunk 1]
```diff
@@ -105,3 +105,16 @@ AC members can be excluded from the AC through an unanimous vote by the other AC
 When excluding a member, a reason has to be provided, such as inactivity, mis-behaviour, etc. and the excluded member must be given the chance to publicly respond to this.
 
 The openHAB Foundation's General Assembly can request that the AC is dissolved and has to be newly appointed.
+
+### Architecture Council Engagement
+
+Any member of the AC can generate a proposal for a decision to be made. 
+They do so by opening a topic in the [dedicated Architecutre category](https://community.openhab.org/c/development/architecture) on the forum.
```
Comment (openhab-5iver) [inline developers/contributing/governance.md]: Typo... Architecture

Comment (Confectrician) [inline developers/contributing/governance.md]:
```suggestion
They do so by opening a topic in the [Architecture category](https://community.openhab.org/c/development/architecture) on the forum.
```

File: developers/contributing/governance.md [hunk 2]
```diff
@@ -105,3 +105,16 @@ AC members can be excluded from the AC through an unanimous vote by the other AC
 When excluding a member, a reason has to be provided, such as inactivity, mis-behaviour, etc. and the excluded member must be given the chance to publicly respond to this.
 
 The openHAB Foundation's General Assembly can request that the AC is dissolved and has to be newly appointed.
+
+### Architecture Council Engagement
```
Comment (kaikreuzer) [inline developers/contributing/governance.md]: Sleeping about this and seeing the actual effect of such a process, I think this goes against my original reason for establishing the AC, which is to remove bottlenecks and speed up decision making. A "government" does not need to do a people vote on every decision. This should at max be a case by case decision, if there's any doubt. So I'd suggest to leave not add this section as it effectively is not "a more detailed description", but rather a change in the process.

**Comment by kaikreuzer** [comment]
I've included the first change in https://github.com/openhab/openhab-docs/pull/898, so this PR can be closed.

**Pull request closed without merging**